cmd/overlay/main.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/kademlia"
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/storage/redis"
+)
+
+var (
+	redisAddress  string
+	redisPassword string
+	db            int
+)
+
+func main() {
+	initalizeFlags()
+	// TODO(coyle): context.WithCancel
+	// TODO(coyle): metrics
+	ctx := context.Background()
+	// bootstrap network
+	kad := kademlia.Kademlia{}
+
+	kad.Bootstrap(ctx)
+	// bootstrap cache
+	cache, err := redis.NewOverlayClient(redisAddress, redisPassword, db, kad)
+	if err != nil {
+		// TODO(coyle): handle error
+	}
+	if err := cache.Bootstrap(ctx); err != nil {
+		// TODO(coyle): handle error
```

Comment:
i know these are TODOs and that's fine, but i'd rather merge code that has `panic("TODO")` instead of just a comment just so no one forgets that this case is unhandled.